server/threads/sql/index.ts

```diff
@@ -30,14 +27,13 @@ const getRoleByExternalIdAndBoardId = `
     LEFT JOIN board_user_roles bur
       ON roles.id = bur.role_id
     LEFT JOIN realm_user_roles rur
```

Comment:
Honestly these queries would be easier to read if we didn't use these shortenings like `rur` and `bur`.

Reply:
Seconding this

Reply:
done